bitrise.yml

```diff
@@ -0,0 +1,85 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - DEPLOY_REPO_URL: $DEPLOY_REPO_URL
+  - VAR_REPOSITORY: $VAR_REPOSITORY
+  # Define it (DEPLOY_PAT) in .bitrise.secrets.yml
+  - DEPLOY_PAT: $DEPLOY_PAT
+  # If you want to share this step into a StepLib
+  - BITRISE_STEP_ID: update-gitops-repository
+  - BITRISE_STEP_VERSION: "0.0.1"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-io/bitrise-step-update-gitops-repository.git
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+
+workflows:
+  test:
+    steps:
+    - path::./:
+        title: Test updating files of a GitOps repository
```

Comment:
Given that `update_files_test` is pretty comprehensive (basically an e2e test), do we need this?

Reply:
I was thinking about deleting this actually, but it has it's benefits: `update_files_test` (in conjunction with all the other tests) are covering everything except the actual interaction with Github API (it's mocked). I found it useful to run 1-2 scenarios (ie. direct push and PR opening) manually while also having a whole bunch of automated test cases not relying on an external API.

A bit off, but related: should we run the automated test suite (not this workflow) in Bitrise CI on every change (not sure about the convention for steps)?

Reply:
So basically we could test the github client we're using, right? I believe that should be tested by the maintainer, right? 

I think e2e tests should definitely run for pr-s, but not necessary for all pushes.

Reply:
Well, we are testing if we are using the Github client the right way (ie. a PR body doesn't have to be provided, but the PR's title is a must).

Reply:
But that can be covered by the mock, right? What am I missing here? We can talk this over in person next week too :)